benchmarks/multi_stream_allocations/multi_stream_allocations_bench.cu

```diff
@@ -31,15 +29,18 @@
 
 #include <cuda_runtime_api.h>
 
+#include <benchmark/benchmark.h>
+
 #include <cstddef>
 
 __global__ void compute_bound_kernel(int64_t* out)
 {
   clock_t clock_begin   = clock64();
   clock_t clock_current = clock_begin;
+  auto const million{1'000'000};
 
-  if (threadIdx.x == 0) {
-    while (clock_current - clock_begin < 1000000) {
+  if (threadIdx.x == 0) {  // NOLINT(readability-static-accessed-through-instance)
```

Comment:
wat. Is clang-tidy going to complain about every usage of `threadIdx.x`?

Reply:
I am unsure about this. We can probably disable the rule for .cu files. In RMM there are so few .cu files that NOLINT isn't really a problem.